lib/Dialect/FIRRTL/FIRRTLReductions.cpp

```diff
@@ -13,6 +13,7 @@
 #include "circt/Dialect/FIRRTL/FIRRTLUtils.h"
 #include "circt/Dialect/FIRRTL/Passes.h"
 #include "circt/Reduce/ReductionUtils.h"
+#include "circt/Support/Namespace.h"
```

Comment:
What makes this necessary?

Reply:
The file uses `circt::Namespace` on L1045.

Reply:
I guess previously this was satisfied by a transitive include.